app/models/rules_of_origin/scheme_set.rb

```diff
@@ -0,0 +1,65 @@
+module RulesOfOrigin
+  class SchemeSet
+    class_attribute :current
```

Comment:
Do we want to avoid inheritance quirks with this way of setting class setters/getters? I guess it's the normal rails way to do this now :thinking:

Reply:
This is specifically intended to be for the full lifecycle of the application - as opposed to anything which changes

Could do something like assigning a constantant instead?

Reply:
I actually don't mind. The effect is clear. It's just a bit new to me